Mathlib/Topology/UnitInterval.lean

```diff
@@ -67,12 +67,20 @@ instance : BoundedOrder I := Set.Icc.boundedOrder zero_le_one
 
 lemma univ_eq_Icc : (univ : Set I) = Icc (0 : I) (1 : I) := Icc_bot_top.symm
 
+@[norm_cast]
 theorem coe_ne_zero {x : I} : (x : ℝ) ≠ 0 ↔ x ≠ 0 :=
   not_iff_not.mpr coe_eq_zero
 
+@[norm_cast]
 theorem coe_ne_one {x : I} : (x : ℝ) ≠ 1 ↔ x ≠ 1 :=
   not_iff_not.mpr coe_eq_one
 
+@[norm_cast]
+lemma coe_pos {x : I} : (0 : ℝ) < x ↔ 0 < x := Iff.rfl
+
+@[norm_cast]
+lemma coe_lt_one {x : I} : (x : ℝ) < 1 ↔ x < 1 := Iff.rfl
```

Comment:
```suggestion
@[simp, norm_cast] lemma coe_pos {x : I} : (0 : ℝ) < x ↔ 0 < x := Iff.rfl
@[simp, norm_cast] lemma coe_lt_one {x : I} : (x : ℝ) < 1 ↔ x < 1 := Iff.rfl
```
Same above

Reply:
If you mean with 'same above' the theorems `coe_ne_zero` and `coe_ne_one` then those are according to the linter not in simp-normal form. Or am I misunderstanding something?

Reply:
`@unitInterval.coe_ne_zero` simp can prove this: `by simp only [@ne_eq, @Set.Icc.coe_eq_zero]`
`@unitInterval.coe_ne_one` simp can prove this: `by simp only [@ne_eq, @Set.Icc.coe_eq_one]`